packages/components/src/Filter/FilterSelect/FilterSelect.tsx

```diff
@@ -4,19 +4,19 @@ import { HiddenSelect, useSelect } from '@react-aria/select'
 import { useSelectState, type SelectProps as AriaSelectProps } from '@react-stately/select'
 import { Filter, FilterContents } from '~components/Filter/Filter'
 import { type FilterButtonProps } from '~components/Filter/FilterButton'
-import { SelectProvider } from '~components/__next__/Select/context'
+import { SelectProvider } from '~components/SingleSelect/context'
```

Comment:
The use of `SingleSelect`context and props throughout our filters does give off a code smell to me.

Not that we have change anything now, but when I was looking into the Select jumping bug yonks ago, the interdependence of these components creates a strange relationship that makes it harder to confidently make changes without risk